opencog/attentionbank/cython/opencog/CMakeLists.txt

```diff
@@ -22,10 +22,9 @@ SET(CYTHON_FLAGS "-f")
 # Cythonizes one .pyx file into a .cpp file
 # Additional arguments are dependencies
 MACRO(CYTHON_ADD_MODULE_PYX name)
```

Comment:
This and the below seem to be cut-n-pastes of one-another. This macro should almost surely be moved to the atomspace repo, `/cmake/OpenCogFunctions.cmake` which is where the macro for guile modules lives. (or `/cmake/OpenCogCython.cmake` or something like that. Maybe rename `OpenCogFunctions.cmake` to `OpenCogGuile.cmake` too ... not now but in some later fix.

Reply:
I thought the same thing. Thanks for the tip for where it should be placed.